tests/unit_tests/test_material.py

```diff
@@ -495,4 +499,22 @@ def test_activity_of_metastable():
     m1.add_nuclide("Tc99_m1", 1)
     m1.set_density('g/cm3', 1)
     m1.volume = 98.9
-    assert pytest.approx(m1.activity, rel=0.001) == 1.93e19
+    assert pytest.approx(m1.get_activity(), rel=0.001) == 1.93e19
+
+
+def test_specific_activity_of_tritium():
+    """Checks that specific and volumetric activity of tritium are correct"""
+    m1 = openmc.Material()
+    m1.add_nuclide("H3", 1)
+    m1.set_density('g/cm3', 1)
+    assert m1.get_activity(normalization='mass') == 355978108155965.9  # [Bq/g]
+    assert m1.get_activity(normalization='mass', by_nuclide=True) == {
+        "H3": 355978108155965.9  # [Bq/g]
+    }
+    # volume is required to calculate total and volumetric activity
```

Comment:
we don't need volume to get Bq/cc only total activity requires volume to be set. We should also run these tests with density and volume not equal to 1 or 10 to make sure we're catching errors in the actual function

Reply:
I've reordered this test in commit https://github.com/openmc-dev/openmc/pull/2142/commits/e4d54d49a8f92aa5b90bfd2b814b6bcc751c3541 to ensure volume is only set at the last oppertunity